src/Atmos/Model/linear.jl

```diff
@@ -510,3 +510,30 @@ function numerical_flux_first_order!(
 
     parent(fluxᵀn) .-= M * Λ * (M \ Δstate) / 2
 end
+
+function numerical_flux_first_order!(
+    ::LMARSNumericalFlux,
+    balance_law::AtmosLinearModel,
+    fluxᵀn::Vars{S},
+    normal_vector::SVector,
+    state_prognostic⁻::Vars{S},
+    state_auxiliary⁻::Vars{A},
+    state_prognostic⁺::Vars{S},
+    state_auxiliary⁺::Vars{A},
+    t,
+    direction,
+) where {S, A}
+
+    numerical_flux_first_order!(
+        RusanovNumericalFlux(),
```

Comment:
Is Rusanov a good choice here? It doesn't satisfy the contact property.

Reply:
@Zhengyu-Huang @akshaysridhar Is Rusanov ok? Or would central be better?

Reply:
@simonbyrne Linear-acoustics results in the Rusanov expression for this flux method